shared/src/commonMain/kotlin/bav/petus/core/dialog/DialogSystem.kt

```diff
@@ -163,6 +166,14 @@ class DialogSystem(
         }
     }
 
+    private fun getOldChancesOfDeathStringId(pet: Pet): StringId{
+        return when{
```

Comment:
There is no need to repeat `pet.deathOfOldAgePossibility * 100` multiple times, please move it to separate `val chance = pet.deathOfOldAgePossibility*100` and reuse.